### DISCUSSION TITLE
[Feature] Search for Cities and Countries in the Search Bar

### DISCUSSION THREAD

**Discussion by iwannet** [immich-app/immich#14609] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be great if, in addition to filtering by country or city, we could also search directly for them in the search bar. This would make it much easier to find specific locations, especially when you have a large number of cities or countries. 
Typing the name of the city or country would quickly bring up relevant results, saving time compared to manually filtering through lists. 

Thanks for considering!

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by oustedaisse** [comment]
Oh yes, I have exactly the same request !
On "App-Which-Must-Not-Be-Named" ;) , it's possible to use search bar to search cities, countries, people etc, with active auto-completition. It's a fastest way to find everything, instead of use dropdowns menus / filtering.
Immich search bar seems to be limited to machine learning content : I think it could be better :)

Thanks for amazing job on Immich !

**Comment by warreth** [comment]
https://github.com/immich-app/immich/pull/25217

**Comment by bo0tzz** [comment]
#7264
